docs/verification.md

```diff
@@ -0,0 +1,58 @@
+# Content Verification in eStargz
+
+The goal of the verifiability in eStargz is to make all files' metadata and contents in an eStargz layer verifiable.
```

Comment:
Definition of "verify"?

Reply:
Through the series of commits related #114, what I aimed to was enabling:

"ensuring the downloaded metadata and contents of all files are the expected ones, based on the calculated OCI digests"
And I called it "verification" so far. 

So the first line should be:
```
The goal of the content verification in eStargz is to ensure the downloaded metadata and contents of all files are the expected ones, based on the calculated OCI digests.
```

Please tell me if I'm missing something or there is something to be added/fixed in the definition and/or the implementation.

Reply:
And thanks for the comment! :pray: